keps/sig-node/4369-allow-special-characters-environment-variable/kep.yaml

```diff
@@ -0,0 +1,43 @@
+title: Allow special characters environment variable
+kep-number: 4369
+authors:
+  - "@HirazawaUi"
+owning-sig: sig-node
+participating-sigs:
+  - sig-api-machinery
+status: provisional
+creation-date: 2023-12-21
+reviewers:
+  - "@liggitt"
+  - "@thockin"
+  - TBD
+approvers:
+  - "@liggitt"
+  - "@thockin"
+  - TBD
+
+see-also: []
+replaces: []
+
+# The target maturity stage in the current dev cycle for this KEP.
+stage: alpha
+
+# The most recent milestone for which work toward delivery of this KEP has been
+# done. This can be the current (upcoming) milestone, if it is being actively
+# worked on.
+latest-milestone: "v1.30"
+
+# The milestone at which this feature was, or is targeted to be, at each stage.
+milestone:
+  alpha: "v1.30"
```

Comment:
Not sure these are right for a `provisional` KEP.

Reply:
I will modify it if it is not implemented in 1.30.

Reply:
Provisional KEPs are not supposed to have code merging, versus `implementable`
The idea being, provisional KEPs may have part of an idea merged to iterate on but aren't agreed to implement yet. An implementable KEP would be expected to have stronger approval and would have a target release for implementing. https://github.com/kubernetes/enhancements/blob/master/keps/sig-architecture/0000-kep-process/README.md#kep-workflow